### DISCUSSION TITLE
Uninstall to start fresh.

### DISCUSSION THREAD

**Discussion by clive7391** [1N3/Sn1per#351] [Q&A] [answered]
I have 9 pro and all the add-ons. Things seem to be a little mismatched and unable to upgrade past 9.

Is there a way to completely remove everything and start over fresh and add licenses?

**Comment by 1N3** [comment, marked as answer]
Yes, to completely remove Sn1per Pro and re-install, you will need to do the following as 'root':

```
cd /sniper/
./install_pro.sh uninstall
./uninstall.sh
```
Next, re-run the activation.sh script from your activation email for Sn1per Professional to re-install Sn1per Pro and that should do the trick. Keep in mind that you will lose ALL loot/workspace data stored by Sn1per, so be sure to backup your /sniper/loot/ directory if you want to restore it.
